packages/bff/src/routers/catalogRouter.ts

```diff
@@ -184,8 +184,8 @@ const catalogRouter = (
         const ctx = fromBffAppContext(req.ctx, req.headers);
         try {
           await catalogService.deleteDraft(
-            req.params.eServiceId,
-            req.body,
+            unsafeBrandId(req.params.eServiceId),
+            unsafeBrandId(req.params.descriptorId),
```

Comment:
Here was the bug: we passed `req.body` instead of  `req.params.descriptorId`.
I also used this PR to align the branded types used as parameters in the catalog service.